packages/dropdown/src/Dropdown.tsx

```diff
@@ -6,104 +6,107 @@ import { DropdownProps, SelectOptions } from './Dropdown.types';
 
 import styles from './styles/default.module.css';
 
-function Dropdown({
-  Component = 'div',
-  value,
-  onChange,
-  options,
-  disabled,
-  dataTestId,
-  onBlur,
-  onClick,
-  fullWidth,
-  ...restProps
-}: DropdownProps) {
-  const [stateValue, setStateValue] = useState<SelectOptions | undefined>(
-    value,
-  );
+const Dropdown = React.forwardRef<HTMLSelectElement, DropdownProps>(
+  (
+    {
+      Component = 'div',
+      value,
+      onChange,
+      options,
+      disabled,
+      dataTestId,
+      onBlur,
+      onClick,
+      fullWidth,
+      ...restProps
+    },
+    ref,
+  ) => {
+    const [stateValue, setStateValue] = useState<SelectOptions | undefined>(
```

Comment:
with something like this, IMO they could be more descriptive:
`stateValue` -> `selectedOption`
`setStateValue` -> `setSelectedOption`

Reply:
resolved